### PR TITLE
Re-write CodeCommit instructions

### PR DESCRIPTION
**Pull request by ChrisBAshton**
1. Provides new step by step guidance specifically for 2nd line drills, to minimise the risk of mistakes being made
1. Removes 'repository-mirroring.html' file which contained useful information that was easy to miss. Consolidates that info into this main doc instead.
1. Sets up redirect from 'repository-mirroring.html' to 'github-unavailable.html' in case people have it bookmarked.
1. Greatly simplifies the actual steps involved in pushing content to CodeCommit.
1. Greatly simplifies the workflow around release tagging, on the basis that GitHub outages or sensitive PRs are short-lived in nature, so can be reasonably managed with hotfix branches that can then be merged upstream later and automatically given their own release tag as normal.

### CODE REVIEW

File: source/manual/github-unavailable.html.md [hunk 1]
```diff
@@ -6,125 +6,117 @@ layout: manual_layout
...
 
-1. Review the pull request on AWS CodeCommit through the [AWS Console](https://eu-west-2.console.aws.amazon.com/codesuite/codecommit/repositories?region=eu-west-2) (access to GOV.UK repos must be granted by a GDS AWS administrator).
+Once you've [pushed your hotfix branch to CodeCommit](#making-changes-to-code-in-aws-codecommit-before-deployment), you should:
 
-1. Create a release tag manually in git. This should follow the standard format
-   `release_X`. Tag the branch directly instead of merging it.
+1. Create a pull request in the CodeCommit UI.
+1. Ask another developer to review it as a sense check.
+1. With the branch reviewed, you can then [deploy it](#deploy-from-aws-codecommit), specifying your branch name as the `TAG` (in the example above, 'hotfix'). Do this in Integration or Staging first, then manually test the changes.
+1. Once you're confident in your changes, deploy the branch to Production.
```
Comment (maxgds) [inline source/manual/github-unavailable.html.md:76]: In the example given, removing the readme, how would I know it's successfully landed on the target environment?

Comment (ChrisBAshton) [reply source/manual/github-unavailable.html.md:76]: I tested this this morning - SSH'ing into the relevant machine, `cd /var/apps/{REPO_NAME}`, and checking that the README wasn't there.
A more visible example would probably be better, but difficult to give one that would work for any app 🤔

Comment (maxgds) [reply source/manual/github-unavailable.html.md:76]: Does it warrant a mention here? I realise the instructions are for generic "deploy a thing during disaster" scenarios but if trying to drill this I would have hit a bit of a wall at this point.

File: source/manual/github-unavailable.html.md [hunk 2]
```diff
@@ -6,125 +6,117 @@ layout: manual_layout
...
+### Deploying a hotfix from AWS CodeCommit
 
-1. Review the pull request on AWS CodeCommit through the [AWS Console](https://eu-west-2.console.aws.amazon.com/codesuite/codecommit/repositories?region=eu-west-2) (access to GOV.UK repos must be granted by a GDS AWS administrator).
+Once you've [pushed your hotfix branch to CodeCommit](#making-changes-to-code-in-aws-codecommit-before-deployment), you should:
 
-1. Create a release tag manually in git. This should follow the standard format
-   `release_X`. Tag the branch directly instead of merging it.
+1. Create a pull request in the CodeCommit UI.
+1. Ask another developer to review it as a sense check.
+1. With the branch reviewed, you can then [deploy it](#deploy-from-aws-codecommit), specifying your branch name as the `TAG` (in the example above, 'hotfix'). Do this in Integration or Staging first, then manually test the changes.
```
Comment (ChrisBAshton) [inline source/manual/github-unavailable.html.md:75]: In response to https://github.com/alphagov/govuk-developer-docs/pull/3738/files#r1005348807:

```suggestion
1. With the branch reviewed, you can then [deploy it](#deploy-from-aws-codecommit), specifying your branch name as the `TAG` (in the example above, 'hotfix'). Do this in Integration or Staging first, then manually test the changes.
  - In the example above, you can check the README was removed by SSH'ing into the relevant machine and executing `ls /var/apps/{REPO_NAME}` to confirm that the README is absent.
```

File: source/manual/github-unavailable.html.md (hunk 1 above)
Comment (ChrisBAshton) [reply source/manual/github-unavailable.html.md:76]: How does https://github.com/alphagov/govuk-developer-docs/pull/3738/files#r1005754232 sound?

File: source/manual/github-unavailable.html.md (hunk 2 above)
Comment (maxgds) [reply source/manual/github-unavailable.html.md:75]: 👍

**Review by maxgds — APPROVED** [review]
Looks good to me, will be interesting to see it stress tested in later drills.

**Comment by ChrisBAshton** [comment]
I'll hold off merging until @AgaDufrat and @catalinailie have had a chance to follow these instructions in a drill this shift 👍

File: source/manual/github-unavailable.html.md [hunk 3]
```diff
@@ -6,123 +6,118 @@ layout: manual_layout
...
+Once security is disabled, anyone on GDS-trusted IPs will be able to invoke any Jenkins job, including `Deploy_App`. Do not leave security disabled for any longer than necessary.
 
-   ```
-   git remote add aws https://git-codecommit.eu-west-2.amazonaws.com/v1/repos/<app>
-   ```
+See the [Jenkins documentation](https://www.jenkins.io/doc/book/security/securing-jenkins/) for further details.
 
-1. Fetch the AWS upstream by running `git fetch aws`.
-   You'll need to provide a username and password, which you can retrieve from govuk-secrets
-   (pass/2ndline/govuk-repo-mirror/https_git_credentials.gpg).
```
Comment (ChrisBAshton) [inline source/manual/github-unavailable.html.md:35]: I've asked @beccapearce to try these instructions out to see if that's a viable workaround for the Mac issue 🤞

**Pull request merged**